### DISCUSSION TITLE
Replace issue templates with issue forms?

### DISCUSSION THREAD

**Discussion by spenserblack** [o2sh/onefetch#663] [Ideas]
Looks like GitHub is beta testing issue forms, which are more structured than issue templates. Docs are here https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms. [Jest has a decent example of this](https://github.com/facebook/jest/issues/new?assignees=&labels=%3Aspeech_balloon%3A+Question&template=question.yml&title=%5BPlease+read+the+message+below%5D). We might want to use forms instead.

**Comment by o2sh** [comment]
I love it 😃 
If jest is using it, I think we can safely do it too - even if it's still in beta.
